### PR TITLE
Migrate tests f-i to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/test/inspector_integration_test.dart [hunk 1]
```diff
@@ -65,9 +65,9 @@ void main() async {
       await tester.pump(const Duration(seconds: 1));
       final InspectorScreenBodyState state =
           tester.state(find.byType(InspectorScreenBody));
-      final controller = state.inspectorController;
+      final controller = state.inspectorController!;
       while (!controller.flutterAppFrameReady) {
-        await state.inspectorController.maybeLoadUI();
+        await state.inspectorController!.maybeLoadUI();
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/inspector_integration_test.dart:70]: use `controller` here

File: packages/devtools_app/test/inspector_integration_test.dart [hunk 2]
```diff
@@ -412,12 +412,12 @@ void main() async {
       await tester.pumpAndSettle(const Duration(seconds: 1));
       final InspectorScreenBodyState state =
           tester.state(find.byType(InspectorScreenBody));
-      final controller = state.inspectorController;
+      final controller = state.inspectorController!;
       while (!controller.flutterAppFrameReady) {
-        await state.inspectorController.maybeLoadUI();
+        await state.inspectorController!.maybeLoadUI();
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/inspector_integration_test.dart:417]: ditto

File: packages/devtools_app/test/inspector_service_test.dart [hunk 3]
```diff
@@ -373,8 +375,9 @@ void main() async {
         );
 
         // Get selection in the render tree.
-        selection = await group.getSelection(null, FlutterTreeType.renderObject,
-            isSummaryTree: false);
+        selection = (await group.getSelection(
+            null, FlutterTreeType.renderObject,
+            isSummaryTree: false))!;
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/inspector_service_test.dart:380]: tc

File: packages/devtools_app/test/inspector_integration_test.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/test/inspector_integration_test.dart:70]: done

File: packages/devtools_app/test/inspector_integration_test.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/test/inspector_integration_test.dart:417]: done

File: packages/devtools_app/test/inspector_service_test.dart [hunk 4]
```diff
@@ -46,21 +46,21 @@ void main() async {
...
         // the daemon api is now false.
       });
 
       test('createObjectGroup', () async {
         await env.setupEnvironment();
 
-        final g1 = inspectorService.createObjectGroup('g1');
-        final g2 = inspectorService.createObjectGroup('g2');
+        final g1 = inspectorService!.createObjectGroup('g1');
+        final g2 = inspectorService!.createObjectGroup('g2');
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/test/inspector_service_test.dart:63]: done

File: packages/devtools_app/test/inspector_service_test.dart [hunk 5]
```diff
@@ -75,31 +75,31 @@ void main() async {
 
       test('infer pub root directories', () async {
         await env.setupEnvironment();
-        final group = inspectorService.createObjectGroup('test-group');
+        final group = inspectorService!.createObjectGroup('test-group');
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/test/inspector_service_test.dart:78]: done

File: packages/devtools_app/test/inspector_service_test.dart [hunk 6]
```diff
@@ -360,8 +362,8 @@ void main() async {
         expect(nodeInDetailsTree.valueRef, equals(nodeInSummaryTree.valueRef));
 
         await group.setSelectionInspector(nodeInDetailsTree.valueRef, true);
-        var selection = await group.getSelection(null, FlutterTreeType.widget,
-            isSummaryTree: false);
+        var selection = (await group.getSelection(null, FlutterTreeType.widget,
+            isSummaryTree: false))!;
```
[the remark this answers is not in the document]
Comment (polina-c) [reply packages/devtools_app/test/inspector_service_test.dart:366]: done

File: packages/devtools_app/test/inspector_service_test.dart (hunk 3 above)
Comment (polina-c) [reply packages/devtools_app/test/inspector_service_test.dart:380]: done

**Pull request merged**